PYME/Acquire/Hardware/driftTracking.py

```diff
@@ -188,6 +188,19 @@ def _setRefN(self, frame_data, N):
         self.calFTs[:,:,N] = ifftn(ref)
         self.calImages[:,:,N] = ref*self.mask
         
+    def set_subroi(self, position):

```

Comment:
```suggestion
    def set_subroi(self, bounds):
```

technically more than a position ...